src/actions/dot/sudoAs.ts

```diff
@@ -61,79 +67,118 @@ export const sudoAs = async (cmd: Command) => {
     throw Error("Failed to pass in a method to get the address.");
   }
 
-  if (sudoKey.address !== (await api.query.sudo.key()).toString()) {
-    console.log(sudoKey.address);
+  const polkadotPrefixAddr = encodeAddress(sudoKey.address, 0);
+
+  if (polkadotPrefixAddr !== (await api.query.sudo.key()).toString()) {
+    console.log(polkadotPrefixAddr);
     console.log((await api.query.sudo.key()).toString());
     throw Error("This is not the secret for the Sudo key.");
   }
 
-  const accountData = await api.query.system.account(sudoKey.address);
+  const accountData = await api.query.system.account(polkadotPrefixAddr);
   const startingNonce = accountData.nonce.toNumber();
 
   let index = 0;
   try {
     for (const entry of csvParsed) {
       if (!entry) continue;
-
       const { method, source, args } = entry;
       const [s,m] = method.split('.');
 
-      const proposal = api.tx[s][m](...args);
-      const nonce = Number(startingNonce) + index;
-
-      const era = createType(
-        api.registry,
-        "ExtrinsicEra",
-        new GenericImmortalEra(api.registry)
-      );
-
-      const logString = `Sending transaction ${s}::${m} from ${source} with sudo key ${sudoKey.address} and nonce: ${nonce}.`;
-      console.log(logString);
-
-      const thisIndex = index;
-
-      const unsub = await api.tx.sudo
-        .sudoAs(source, proposal)
-        .signAndSend(
-          sudoKey,
-          { blockHash: api.genesisHash, era, nonce },
-          (result) => {
-            const { status } = result;
-
-            console.log("Current status is", status.type);
-            fs.appendFileSync(
-              "sudAs.hashes.log",
-              logString + "\n" + "Current status is" + status.type + "\n"
-            );
-
-            if (status.isFinalized) {
-              console.log(
-                `Transaction included at blockHash ${status.asFinalized}`
-              );
-              fs.appendFileSync(
-                "sudoAs.hashes.map.log",
-                `${logString}\t${status.asFinalized.toString()}\t${
-                  prevHashesParsed[thisIndex]
-                }\n`
-              );
-              fs.appendFileSync(
-                "sudoAs.hashes.log",
-                logString + "\n" + status.asFinalized.toString() + "\n"
-              );
-              // Loop through Vec<EventRecord> to display all events
-              // events.forEach(({ phase, event: { data, method, section } }) => {
-              //   fs.appendFileSync(
-              //     "sudoAs.hashes.log",
-              //     `\t' ${phase}: ${section}.${method}:: ${data}\n`
-              //   );
-              // });
-              unsub();
-            }
+      let proposal;
+
+      switch (m) {
+        case 'claimAttest': 
+          try {
+            const unsub = await api.tx.claims[m](source, ...args)
```

Comment:
```suggestion
            const unsub = await api.tx.claims[m](source, ...args)
```

this should be the `claimAttest` transaction sent by sudoAs.